libs/blocks/graybox-promote/components/graybox-promote-lit.js

```diff
@@ -32,7 +32,7 @@ const KEYS = {
   },
 };
 
-const TELEMETRY = { application: { appName: 'Adobe Graybox Promote' } };
+// const TELEMETRY = { application: { appName: 'Adobe Graybox Promote' } };
```

Comment:
If it's no longer needed, let's delete it.

Reply:
I'll need it for my next ticket. I commented it out to avoid the eslint check failing.